src/coreclr/src/gc/vxsort.cpp

```diff
@@ -0,0 +1,282 @@
+
```

Comment:
License header?

It may need an entry in
https://github.com/dotnet/runtime/blob/master/THIRD-PARTY-NOTICES.TXT

Reply:
I don't mind adding whatever license you feel is friendliest to the original

Reply:
In the meantime, I've licensed under MIT license:
https://github.com/damageboy/gcsort/blob/master/LICENSE